src/ducks/ui/listevisning-selectors.ts

```diff
@@ -192,12 +180,33 @@ export function getMuligeKolonner(filtervalg: FiltervalgModell, oversiktType: Ov
         .concat(addHvis(Kolonne.TOLKEBEHOV_SPRAAK, tolkBehovErValgt()))
         .concat(addHvis(Kolonne.TOLKEBEHOV_SIST_OPPDATERT, tolkBehovErValgt()))
         .concat(addHvis(Kolonne.AVVIK_14A_VEDTAK, avvik14aVedtakErValgt()))
+        .concat(
+            addHvis(
+                Kolonne.VURDERINGSFRIST_YTELSE,
+                featureAAPkolonne &&
+                    (filtervalg.ytelse === AAP_YTELSE ||
+                        filtervalg.ytelse === AAP_YTELSE_MAXTID ||
+                        filtervalg.ytelse === AAP_YTELSE_UNNTAK)
+            )
+        )
+        .concat(addHvis(Kolonne.TYPE_YTELSE, featureAAPkolonne && filtervalg.ytelse === AAP_YTELSE))
+        .concat(addHvis(Kolonne.VEDTAKSPERIODE, filtervalg.ytelse === AAP_YTELSE_UNNTAK))
+        .concat(addHvis(Kolonne.RETTIGHETSPERIODE, filtervalg.ytelse === AAP_YTELSE_MAXTID))
         .concat(
             addHvis(
                 Kolonne.VEILEDER,
                 oversiktType === OversiktType.enhetensOversikt && !filtervalg.ferdigfilterListe.includes(MOTER_IDAG)
             )
         )
+        .concat(
+            addHvis(Kolonne.VEDTAKSPERIODE, filtervalg.ytelse === AAP_YTELSE || filtervalg.ytelse === AAP_YTELSE_MAXTID)
```

Comment:
Går det ann å filtrere på både UNNTAK og enten AAP_YTELSE eller MAXTID samtidig vil du potensielt få opp to vedtaksperioder. HAr du testet no rundt det? Stemmer det eller burde man hatt med no " && filtervalg.ytelse !== AAP_YTELSE_UNNTAK her? 

Det at det kan dukke opp "to like" kolonner dersom man kan kombinere filtere gjelder da kanskje også på kolonnen under (RETTIGHETSPERIODE).

Reply:
Aha, nei det skal ikke være mulig, alle tre er en del av fieldset/radiogroup "Dagpenger, AAP og tiltakspenger", så kun en av gangen!